embrace-android-sdk/src/main/cpp/unwinders/stack_unwinder.cpp

```diff
@@ -44,6 +44,9 @@ static inline void emb_copy_frame_data(unwindstack::AndroidUnwinderData &android
         data->offset = map_info->offset();
         data->flags = map_info->flags();
         emb_strncpy(data->full_name, map_info->GetFullName().c_str(), EMB_FRAME_STR_SIZE);
+
+        // calculate line number
+        data->line_num = data->pc - data->start;
```

Comment:
isn't  `line_num` meant to refer to the symbolicated line number? this looks like it's computing the module address offset